src/mcts/node.h

```diff
@@ -225,7 +270,8 @@ class Node {
   EdgeList edges_;
 
   // 8 byte fields.
-  // Pointer to a parent node. nullptr for the root.
+  // Pointer to a parent node. nullptr for the root of tree, 
+  // root of tree != search->root_node_.
```

Comment:
That's a false statement. root of `tree == search->root_node_` in startpos.
Better something like: "Note that root of the tree is game's startpos and not root of the search."